chapters/publication.tex

```diff
@@ -99,91 +181,265 @@ \section{Collaborating on academic writing}
+In all other cases, you should contact an appropriate data catalog
+to determine what privacy and licensing options are available.
+
+Make sure you have a clear understanding of the rights associated with the data release
+and communicate them to any future users of the data.
+You must provide a license with any data release.\sidenote{
+  \url{https://iatistandard.org/en/guidance/preparing-organisation/organisation-data-publication/how-to-license-your-data/}}
+This document need not be extremely detailed,
+but it should clearly communicate to the reader what they are allowed to do with your data and
+how credit should be given and to whom in further work that uses it.
+Keep in mind that you may or may not own your data,
+depending on how it was collected,
+and the best time to resolve any questions about these rights
+is at the time that data collection or transfer agreements are signed.
+Even if you cannot release data immediately or publicly,
+there are often options to catalog or archive the data without open publication.
+These may take the form of metadata catalogs or embargoed releases.
+Such setups allow you to hold an archival version of your data
+which your publication can reference,
+as well as provide information about the contents of the datasets
+and how future users might request permission to access them
+(even if you are not the person who can grant that permission).
+They can also provide for timed future releases of datasets
+once the need for exclusive access has ended.
+
+Data publication should release the dataset in a widely recognized format.
+While software-specific datasets are acceptable accompaniments to the code
+(since those precise materials are probably necessary),
+you should also consider releasing generic datasets
+such as CSV files with accompanying codebooks,
+since these will be re-adaptable by any researcher.
+Additionally, when possible, you should also release
+the data collection instrument or survey questionnaire used to gather the information
+so that readers can understand which data components are
+collected directly in the field and which are derived.
+You should provide a clean version of the data
+which corresponds exactly to the original database or questionnaire
+as well as the constructed or derived dataset used for analysis.
+Wherever possible, you should also release the code
+that constructs any derived measures,
+particularly where definitions may vary,
+so that others can learn from your work and adapt it as they like.
+
+\subsection{Publishing code for replication}
+
+Before publishing your code, you should edit it for content and clarity
+just as if it were written material.
+The purpose of releasing code is to allow others to understand
+exactly what you have done in order to obtain your results,
+as well as to apply similar methods in future projects.
+Therefore it should both be functional and readable.
+Code is often not written this way when it is first prepared,
```

Comment:
let's refer back to our chapter on coding as a team, say that if you've followed the steps previously outlined this is straightforward

Reply:
77a6fde65fa985a525f7f498756ec1b0071582b2